samples/RUNBOOK.md

```diff
@@ -0,0 +1,353 @@
+# Bottlerocket Test System Sample Files
+
+These files contain templates that can be populated using `sed` commands and run with the `cli` tool.
+Examples of how to populate each one of these files can be found below. Each codeblock starts with a small number of variables that need to be populated before running the block, followed by a number of variables that contain default values (but can be modified by the user). The final `cat` command will print the populated file to stdout.
+
+## EKS
+
+The files in `eks` are meant to be run on an EKS test cluster. You can create a new cluster using the `eksctl` tool.
+
+### ecs-migration-test
+
+```bash
+export VARIANT=
+export ARCHITECTURE=
+
+export ECS_TEST_AGENT_IMAGE_URI="public.ecr.aws/bottlerocket-test-system/ecs-test-agent:v0.0.5"
```

Comment:
How can we make sure this aligns with the latest released version?

Reply:
Can we put all of the agent images together?

Reply:
Aside from adding a `latest` tag, we might be able to use the `aws ecr` CLI to get the catalog from public.ecr.aws and figure out what the latest version is.